src/main/java/hudson/plugins/emailext/plugins/content/FailedTestsContent.java

```diff
@@ -145,4 +156,13 @@ private int outputTest(StringBuilder buffer, TestResult failedTest,
     private String getLineBreak() {
         return escapeHtml ? "<br/>" : "\n";
     }
+
+    private List<TestResult> filterTests(List<? extends TestResult> failedTests, String regexPattern) {
+        List<TestResult> filteredTests = failedTests.stream().collect(Collectors.toList());
+        if(regexPattern.length() != 0) {

```

Comment:
If you remove the above line, if no pattern is specified, you may return the `failedTests` as is.